specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2023-10-01/machineLearningServices.json

```diff
@@ -2373,6 +2373,9 @@
           "$ref": "../../../../../common-types/resource-management/v3/managedidentity.json#/definitions/ManagedServiceIdentity",
           "description": "The identity of the resource."
         },
+        "kind": {
```

Comment:
I'd like to double confirm that you're adding `kind` to the top level? Should it be under the `properties`?

And consider model it as an enums if the property has a finite set of values?
refs: https://armwiki.azurewebsites.net/api_contracts/guidelines/openapi.html#oapi015-avoid-modelling-a-property-as-string-type-when-it-should-be-an-enums

Reply:
nope, this is on the top level and it's been in our preview version for a while, we just need to bring this to stable, the implementation of this kind property has already go over ARM review before

Reply:
Thanks for the clarification, would you please share me the previous PR that ARM reviewed?

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/22414

should be this one